src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -0,0 +1,1482 @@
+            }
+        }
+
+        [Fact]
+        public async Task ActivatePolicyRule()
+        {
+            var client = TestClient.Create();
+            var guid = Guid.NewGuid();
+
+            var policy = new PasswordPolicy()
+            {
+                // Name has a maximum of 50 chars
+                Name = $"Default policy {guid}".Substring(0, 50),
+                Type = PolicyType.Password,
+                Status = "ACTIVE",
+                Description = "The default policy applies in all situations if no other policy applies.",
+            };
+
+            var createdPolicy = await client.Policies.CreatePolicyAsync(policy);
+
+            IPasswordPolicyRule policyRule = new PasswordPolicyRule()
+            {
+                Name = $"Password Policy Rule {guid}".Substring(0, 50),
+                Type = PolicyType.Password,
+                Conditions = new PasswordPolicyRuleConditions()
+                {
+                    People = new PolicyPeopleCondition()
+                    {
+                        Users = new UserCondition()
+                        {
+                            Exclude = new List<string>(),
+                        },
+                    },
+                },
+                Actions = new PasswordPolicyRuleActions()
+                {
+                    PasswordChange = new PasswordPolicyRuleAction()
+                    {
+                        Access = "ALLOW",
+                    },
+                },
+            };
+
+            var createdPolicyRule = await client.Policies.AddPolicyRuleAsync(policyRule, createdPolicy.Id);
+
+            try
+            {
+                var retrievedPolicyRule = await client.Policies.GetPolicyRuleAsync(createdPolicy.Id, createdPolicyRule.Id);
+                retrievedPolicyRule.Should().NotBeNull();
+                // Default status
+                retrievedPolicyRule.Status.Should().Be("ACTIVE");
+
+                await retrievedPolicyRule.DeactivateAsync(createdPolicy.Id);
+                retrievedPolicyRule = await client.Policies.GetPolicyRuleAsync(createdPolicy.Id, createdPolicyRule.Id);
+                retrievedPolicyRule.Status.Should().Be("INACTIVE");
+
+                await retrievedPolicyRule.ActivateAsync(createdPolicy.Id);
+                retrievedPolicyRule = await client.Policies.GetPolicyRuleAsync(createdPolicy.Id, createdPolicyRule.Id);
+                retrievedPolicyRule.Status.Should().Be("ACTIVE");
+            }
+            finally
+            {
+                await client.Policies.DeactivatePolicyRuleAsync(createdPolicy.Id, createdPolicyRule.Id);
+                await client.Policies.DeletePolicyRuleAsync(createdPolicy.Id, createdPolicyRule.Id);
+                await client.Policies.DeactivatePolicyAsync(createdPolicy.Id);
+                await client.Policies.DeletePolicyAsync(createdPolicy.Id);
+            }
+        }
+
+        [Fact]
+        public async Task CreateMFAPolicy()
```

Comment:
We need to sort out the Beta conversation on this policy before we merge this (same for openapi PR)